scripts/templates/soccer.wbt.template

```diff
@@ -1,4 +1,4 @@
-#VRML_SIM R2021b utf8
+https://github.com/cyberbotics/webots/wiki/How-to-adapt-your-world-or-PROTO-to-Webots-R2022a
```

Comment:
@Adman I know literally nothing about this but shouldn't this line say `#VRML_SIM R2022a utf8`? In all honesty, I am not convinced Webots would be able to parse this. Perhaps we could add one more line with this link somewhere into the file (say the second line)?